functions/funtions.py

```diff
@@ -8,17 +8,16 @@
 def take_data(
     train_data: pd.DataFrame, test_data: pd.DataFrame, movie_name: pd.DataFrame
 ):
-    """Convert data from Pd.DataFrame to numpyarray
+    """Preprocess data from Pd.DataFrame
 
     Args:
-        train_data (pd.DataFrame): train_data
-        test_data (pd.DataFrame): test_data
-        movie_name_cfg (pd.DataFrame): the name of movies
+        train_data (pd.DataFrame): train data load from "csv" file
+        test_data (pd.DataFrame): test data load from "csv" file
+        movie_name_cfg (pd.DataFrame): the name of movies to recommend load from  "csv" file
 
     Returns:
-        train_set (numpydarray)
-        test_set (numpydarray)
-        true_testset_movies_id (numpydarray)
+        train_set, test_set (numpydarray): Preprocessed data
```

Comment:
```suggestion
        train_set (numpy.array): Preprocessed training data
        test_set (numpy.array): Preprocessed testing data
```
It should be `numpy.array` (recommend), or `ndarray`.
Need update everywhere else too.